charts/sn-platform/templates/broker/_broker.tpl

```diff
@@ -396,4 +396,23 @@ pulsar service domain
 {{- print "" -}}
   {{- end -}}
 {{- end -}}
+{{- end -}}
+
+{{/*
+Default name of builtin connectors
+*/}}
+
+{{- define "pulsar.builtinConnectorName" -}}
+{{- print "builtin-connectors" -}}
+{{- end -}}
+
+{{/*
+Get the name of builtin connector for function mesh
+*/}}
+{{- define "pulsar.functionMeshBuiltinConnectors" -}}
```

Comment:
Actually it's the name of the configmap for builtin connectors, would it be better for the name to be more explicit?

Reply:
Fixed with `pulsar.builtinConnectorsConfigMapName`